scripts/sentry-cli.properties

```diff
@@ -0,0 +1,2 @@
+version = 1.69.1
```

Comment:
We can likely bump to https://github.com/getsentry/sentry-cli/releases/tag/1.74.4

Reply:
wanted to let auto-update CI do that :)

Reply:
but wait, there're no good tests in this repo... so bumping to v2.0 should be done with manual testing...

OK, updating to 1.74.4 now
```suggestion
version = 1.74.4
```